packages/components/src/Dropzone/sub-components/OnDragMessage/OnDragMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Paragraph } from "@kaizen/typography"
+
+export const OnDragMessage = (): JSX.Element => (
```

Comment:
This should be renamed to sound more like a component, as we use the "On" prefix for functions.